pkg/pcr/flow_detection.go

```diff
@@ -57,7 +144,7 @@ func isCBnT(fitEntries []fit.Entry) (bool, error) {
 		case *fit.EntryKeyManifestRecord:
 			data, err := fitEntry.ParseData()
 			if data == nil {
-				return false, fmt.Errorf("unable to parse KeyManifest policy record: %w", err)
+				return false, fmt.Errorf("unable to parse KeyManifest policy record: %v", err)
```

Comment:
why? :)

Reply:
I thought that it is a typo, checked the docs, %w is now used for wrapping, will revert this line, thanks!